### PR TITLE
examples: add new qemu x86 FSP examples

### PR DESCRIPTION
**Pull request by rizlik**
adding the necessary script to create FSP binaries to boot qemu x86 q35 machine from the resect vector.
It uses the wolfBoot PR: 
https://github.com/wolfSSL/wolfBoot/pull/274

### CODE REVIEW

File: qemu-x86-fsp-linux/build_qemu_fsp.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
```
Comment (danielinux) [inline qemu-x86-fsp-linux/build_qemu_fsp.sh]: please change to `#!/bin/bash`

File: qemu-x86-fsp-linux/build_and_run_example.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+WOLFBOOT_REPO=${WOLFBOOT_REPO:-"https://github.com/wolfSSL/wolfBoot.git"}
+
+set -e
+
+./build_qemu_fsp.sh
+(cd linux-image && ./compile_linux.sh)
+git clone "$WOLFBOOT_REPO" wolfBoot
```
Comment (danielinux) [inline qemu-x86-fsp-linux/build_and_run_example.sh]: Please use the existing submodule in `../wolfboot` instead

**Review by danielinux — CHANGES_REQUESTED** [review]

**Comment by rizlik** [comment]
addressed comment + fixed error in edk2 build system .map file parser

**Comment by dgarske** [comment]
@rizlik  What's the status of this PR? Is it ready to review/merge?

**Comment by rizlik** [comment]
This is now a stale PR, a working example of FSP running on QEMU can be found in the main wolfBoot repo, see ```./tools/scripts/x86_fsp/qemu/test_qemu.sh``` in https://github.com/wolfSSL/wolfBoot/

**Pull request closed without merging**